### PR TITLE
Libxext 1.3.4 => 1.3.6

### PR DESCRIPTION
**Pull request by uberhacker**
Removed i686 since libx11 is no longer supported.  All tests passed.

### CODE REVIEW

File: packages/libxext.rb [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
   })
 
   depends_on 'llvm16_lib' => :build
+  depends_on 'xmlto' => :build
+  depends_on 'glibc' # R
+  depends_on 'libx11' # R
+  depends_on 'libxau' # R
+  depends_on 'libxcb' # R
+  depends_on 'libxdmcp' # R
+  depends_on 'xorg_proto' # R
```
Comment (satmandu) [inline packages/libxext.rb:26]: Alphabetize the depends?

**Review by satmandu — APPROVED** [review]

File: packages/libxext.rb (hunk 1 above)
Comment (uberhacker) [reply packages/libxext.rb:26]: Done

File: packages/libxext.rb [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
+     armv7l: '2f74453c3b6d4dee2cd194c59134e0fd14f1dc3fcd240edfbd2976021de0c27d',
+     x86_64: '99bd31ebf5bf87c2587ac10bcafc7719a70629660dfb366b0362604a9228d6a1'
   })
 
+  depends_on 'glibc' # R
+  depends_on 'libx11' # R
+  depends_on 'libxau' # R
+  depends_on 'libxcb' # R
+  depends_on 'libxdmcp' # R
   depends_on 'llvm16_lib' => :build
```
Comment (satmandu) [inline packages/libxext.rb:24]: Oops! This should be updated too to `llvm18_lib`

**Review by satmandu — APPROVED** [review]

File: packages/libxext.rb (hunk 2 above)
Comment (uberhacker) [reply packages/libxext.rb:24]: That would require new binaries, right?

Comment (satmandu) [reply packages/libxext.rb:24]: If getrealdeps.rb doesn't complain, then no rebuild is necessary.

Comment (uberhacker) [reply packages/libxext.rb]: Rebuilt binaries anyway to make sure it works.

**Review by satmandu — APPROVED** [review]

**Pull request merged**